README.md

```diff
@@ -135,6 +135,7 @@ cd ~/ros2_mara_ws && colcon build --merge-install --packages-skip individual_tra
 ```bash
 source /opt/ros/melodic/setup.bash
 cd ~/ros2_mara_ws && colcon build --merge-install --packages-select individual_trajectories_bridge
+sed -i 's/melodic/crystal/g' ~/ros2_mara_ws/install/setup.bash
```

Comment:
This sed replace all the apparitions of melodic to crystal, is this right?

Reply:
Yes. It actually only appears once in the setup.bash file. But I can be more specific to point just to the "melodic" I want if necessary.

Reply:
LGTM then.